java/fury-core/src/main/java/org/apache/fury/util/StringUtils.java

```diff
@@ -249,4 +265,30 @@ public static String lowerCamelToLowerUnderscore(String lowerCamel) {
 
     return builder.toString();
   }
+
+  public static boolean isLatin(char[] chars, int charArrayOffset) {
```

Comment:
We might also add an interface like this

```
public static boolean isLatin(String) {}
```

Reply:
A `StringUtils` interface?